e2e-tests/gatsby-static-image/cypress/integration/native-lazy-loading.js

```diff
@@ -0,0 +1,35 @@
+describe(`gatsby-plugin-image / native lazy loading`, () => {
+  // /!\ Make sure to run this one using a real build: not in develop
```

Comment:
It would be nice if we could check for this, though I'm unsure how best to do it. Could be as simple as checking we're not running on port 8000!